contrib/snap/snapcraft.yaml

```diff
@@ -91,6 +91,7 @@ parts:
                        -Dpython=/usr/bin/python3,
                        -Dplugin_flashrom=disabled,
                        -Dplugin_powerd=disabled,
+                       -Dplugin_uefi_dbx_snapd=enabled,
```

Comment:
Since it's a meson tri state feature with no tests to automatically disable from host or missing deps does this need to be hard enabled?

Reply:
Good point, let me look into it.